documentation/modules/security/proc-installing-your-own-ca-certificates.adoc

```diff
@@ -137,7 +104,6 @@ kubectl annotate secret _CA-KEY-SECRET_ strimzi.io/ca-key-generation=_CA-KEY-GEN
 * Annotation `strimzi.io/ca-cert-generation=_CA-CERTIFICATE-GENERATION_` defines the generation of a new CA certificate.
 * Annotation `strimzi.io/ca-key-generation=_CA-KEY-GENERATION_` defines the generation of a new CA key.
 +
-If you are replacing CA certificates automatically generated by the Cluster Operator, use the next higher incremental value from the existing annotation and follow the xref:proc-replacing-your-own-private-keys-{context}[replacing CA keys procedure].
 If there are no CA certificates automatically generated by the Cluster Operator, start from 0 (zero) as the incremental value (`strimzi.io/ca-cert-generation=0`) for your own CA certificate. Set a higher incremental value when you renew the certificates.
```

Comment:
`If there are no CA certificates automatically generated by the Cluster Operator` ... if this is about installing your own cert + key from scratch, this is always true, or? So we could omit it and say to start from 0.

Reply:
Well, I guess you could just be using your own cluster certificates and keeping the clients generated ones?

Reply:
Yeah that's possible but they are following two separated generations. They are not sharing the generation value. So this sentence is not so clear to me imho.

Reply:
Yeah. I think it's clearer without the first sentence.

> Start from 0 (zero) as the incremental value (`strimzi.io/ca-cert-generation=0`) for your own CA certificate. Set a higher incremental value when you renew the certificates.